packages/theming/apple-theme/README.md

```diff
@@ -0,0 +1,10 @@
+# Apple-theme
+
+Code and definitions for creating an Apple Theme for FluentUI React Native. Currently only implemented for macOS. iOS Apple theme progress is tracked at Github #596
+
+Some heuristics followed in this theme:
+
+- For most tokens, we default to prefer the apple system colors, using the FluentUI Apple Palette where it makes sense. These mappings are subject to change as we increment on the design.
+- On macOS, there tends to not be a "hover" state for components such as Button, so those tokens are mapped to be identical to the rest state tokens.
```

Comment:
"rest _of the_ state tokens"? Or is a `rest state token` some token I don't know of

Reply:
If it's the opposite of a pressed state, usually Apple refers to that as the `normal` state

Reply:
rest state as in normal state. I'll add clarification that this is the "normal" state on apple Platforms